processing/src/test/java/org/apache/druid/segment/TestNullableFloatColumnSelector.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.segment;
+
+import org.apache.druid.common.config.NullHandling;
+
+public class TestNullableFloatColumnSelector extends TestFloatColumnSelector
+{
+
+  private final Float[] floats;
+
+  static {
+    NullHandling.initializeForTests();
+  }
+
+  private int index = 0;
+
+  public TestNullableFloatColumnSelector(Float[] floats)
+  {
+    this.floats = floats;
+  }
+
+  @Override
+  public float getFloat()
+  {
+    if (floats[index] != null) {
+      return floats[index];
+    } else {
+      return NullHandling.ZERO_FLOAT;
```

Comment:
Would you please fix here too?

Reply:
@jihoonson, I changed it here in the previous commit: https://github.com/apache/druid/commit/e2639e571c40069828a24be815eb67670f5aaf1e

But then the CI Build Sql compatibility test failed for the Float case, please see below: 
```
[ERROR] testFloatWrappingDimensionSelector(org.apache.druid.segment.WrappingDimensionSelectorTest)  Time elapsed: 0.008 s  <<< ERROR!
java.lang.UnsupportedOperationException
```
I assumed calling this was ok for `float`, but not `long` and `double` types. Please correct me if I am wrong.

Reply:
I'm not sure whether float is special or not. Would you point out the doc you read?

Reply:
@jihoonson, ah, sorry, my bad. You're right - `float` doesn't appear to be a special case, but it was a bug in the test code. The commit above had an [extraneous call](https://github.com/apache/druid/commit/e2639e571c40069828a24be815eb67670f5aaf1e#diff-afef3502218e8fbbfbb4898cc9f88972R99) because of which the test failed.

I pushed the corrected changes in my latest commit. Sorry about the confusion. Could you please take another look?